src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/db/RecallTest.kt

```diff
@@ -92,70 +92,91 @@ class RecallTest {
   }
 
   @Test
-  fun `In Custody Recall with recallNotificationEmailSentDateTime set returns status RECALL_NOTIFICATION_ISSUED`() {
+  fun `In Custody Recall with assessedByUserId set returns status AWAITING_DOSSIER_CREATION`() {
     val recall = recall.copy(
       inCustodyAtBooking = true,
-      recallNotificationEmailSentDateTime = OffsetDateTime.now()
+      assessedByUserId = ::UserId.random().value,
     )
 
-    assertThat(recall.status(), equalTo(Status.RECALL_NOTIFICATION_ISSUED))
+    assertThat(recall.status(), equalTo(Status.AWAITING_DOSSIER_CREATION))
   }
 
   @Test
-  fun `NOT in Custody Recall with bookedByUserId, recallNotificationEmailSentDateTime & warrantReferenceNumber set returns status AWAITING_RETURN_TO_CUSTODY`() {
+  fun `In Custody Recall with assessedByUserId and assessedByUserId set returns status AWAITING_DOSSIER_CREATION`() {
+    val recall = recall.copy(
+      inCustodyAtBooking = true,
+      assessedByUserId = ::UserId.random().value,
+    )
+
+    assertThat(recall.status(), equalTo(Status.AWAITING_DOSSIER_CREATION))
+  }
+
+  @Test
+  fun `NOT In Custody Recall with assessedByUserId and assessedByUserId set returns status ASSESSED`() {
```

Comment:
`with assessedByUserId and assessedByUserId set`?
`returns status ASSESSED`?  -> `returns status ASSESSED_NOT_IN_CUSTODY`.